### DISCUSSION TITLE
Classification Gaussian Naive Bayes

### DISCUSSION THREAD

**Discussion by joejnt** [teamalgoritma/community#390] [Q&A] [answered]
Dear tim Algoritma,

Mohon bantuannya saya mendapatkan permasalahan untuk pemodelan dengan menggunakan Gaussian Naive Bayes, tujuan saya adalah melakukan klasifikasi dengan melakukan prediksi apakah Cabang/Capem akan masuk dalam kategori H/MTH/M/LTM/L dengan vr.prediktor yg tipe nya numerik.

Pada saat saya menjalankan code ini
library(naivebayes)
M <- data_cln %>% 
  select(-c(KODE_CAB, NAMA_CABANG, KATEGORI_CABANG, RISK_ASSESSMENT_2021))
head(M)

Y <- data_cln %>% 
  select(RISK_ASSESSMENT_2021)
glimpse(Y)
head(Y)

gnb <- gaussian_naive_bayes(x = M, y = Y,prior = NULL)
summary(gnb)

hasilnya error. 
Mohon saran dan petunjuknya.

terlampir dataset yang saya gunakan.
[risk assessment.xlsx](https://github.com/teamalgoritma/community/files/9726706/risk.assessment.xlsx)

**Comment by jafarhf1** [comment, marked as answer]
Dear @joejnt,
Bisa tolong dilampirkan untuk hasil errornya seperti apa?
